src/java/com/zimbra/oauth/token/handlers/impl/NextCloudTokenHandler.java

```diff
@@ -70,19 +70,17 @@ public static String refreshAccessToken(Account acct, String client) throws Serv
         final OAuthInfo oauthInfo = new OAuthInfo(new HashMap<String, String>());
         final String refreshToken = OAuth2DataSource.getRefreshToken(dataSource);
 
-        String value = com.zimbra.oauth.token.utilities.LdapOauthHelper.getFirstConfig(Provisioning.A_zimbraOAuthConsumerCredentials,
-                client, acct) ;
-        String indVa [] = value.split(":");
+        String [] values = LdapOauthHelper.loadConfiguration(acct, Provisioning.A_zimbraOAuthConsumerCredentials,
+            client) ;
+        String indVa [] = values[0].split(":");
         final String clientId = indVa[0];
         final String clientSecret = indVa[1];
-        value = LdapOauthHelper.getFirstConfig(Provisioning.A_zimbraOAuthConsumerRedirectUri,
-            client, acct) ;
-        String redirecUri = value.substring(0, value.lastIndexOf(":"));
+        values = LdapOauthHelper.loadConfiguration(acct, Provisioning.A_zimbraOAuthConsumerRedirectUri,
+            client) ;
+        String redirecUri = values[0].substring(0, values[0].lastIndexOf(":"));
         final String clientRedirectUri = redirecUri;
-
-        value = LdapOauthHelper.getFirstConfig(Provisioning.A_zimbraOAuthConsumerAPIScope,client, acct);
-        String nextCloudUrl =value.substring(0, value.lastIndexOf(":"));
-
+        values = LdapOauthHelper.loadConfiguration(acct,  Provisioning.A_zimbraOAuthConsumerAPIScope,client);
```

Comment:
```suggestion
        values = LdapOauthHelper.loadConfiguration(acct,  Provisioning.A_zimbraOAuthConsumerAPIScope, client + "_noop");
```
Revert all of these changes, and just use this instead to fix.